concepts/functions/about.md

```diff
@@ -1,40 +1,167 @@
 # About
 
-Functions in Go are considered [first class citizens][first-class-functions] making them very powerful. They can be used as:
+A function allows you to group code into a reusable unit.
+It consists of the `func` keyword, the name of the function, and a comma-separated list of zero or more parameters and types in round brackets.
+Other possibilities like [methods][concept-methods] and anonymous functions will be covered in other concepts.
```

Comment:
I'm not a fan of referring to other concepts like this upfront. Maybe change these to be [concept widget links](https://exercism.org/docs/building/markdown/widgets) and move this text somewhat later in this document?